### ISSUE TITLE
OpenAPI 3: Support "in: cookie" in securitySchemes

### ISSUE THREAD

**Issue by danielrentz** [LucyBot-Inc/api-spec-converter#184]
According to the spec, it is possible to add cookie-based authentication:
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#securitySchemeObject
https://swagger.io/docs/specification/authentication/cookie-authentication/

But using that with this library results in
>  message: 'expected ''cookie'' to be equals one of Array [ ''query'', ''header'' ]'
>  context: ['#/', '#/components/securitySchemes/SessionCookie']

**Comment by MikeRalphson** [comment]
I have a vague recollection this was a bug fixed some time ago in `swagger2openapi`, so should be fixed as part of #182

**Comment by bobby-brennan** [comment]
Yup, looks like the latest version of oas-validator has `cookie` in the relevant enum. Once #182 is fixed this should be fixed as well.

**Comment by bobby-brennan** [comment]
I believe this is fixed now. Closing.

**Comment by danielrentz** [comment]
Still happens with v2.8.3
